### PR TITLE
Small cleanup after this week feedback

### PR DESCRIPTION
**Pull request by Luthaf**
This is best reviewed one commit at the time =)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,7 +45,6 @@ webpage: https://www.epfl.ch/labs/cosmo/people/)
 git clone https://github.com/cosmo-epfl/chemiscope
 cd chemiscope
 npm install
-npm run download-example-input
```
Comment (rosecers) [inline README.md:48]: Why have we removed this? If you are doing dev it's really important you have the example input. Is it because we're moving the package to be installed more beyond dev purposes?

File: python/chemiscope/input.py [hunk 2]
```diff
@@ -145,9 +145,11 @@ def write_input(path, frames, meta=None, properties=None, cutoff=None):
 
     .. code-block:: python
 
+        import ase
         from ase import io
         import numpy as np
         import sklearn
+        from sklearn import decomposition
```
Comment (rosecers) [inline python/chemiscope/input.py:152]: Why is this necessary? With the import of sklearn, we shouldn't need the explicit decomposition import.

**Review by rosecers — APPROVED** [review]
image

File: README.md (hunk 1 above)
Comment (Luthaf) [reply README.md:48]: No, this is because example input are now automatically downloaded when running `npm start` =)

File: python/chemiscope/input.py (hunk 2 above)
Comment (Luthaf) [reply python/chemiscope/input.py:152]: At least locally, I get this error:

```
>>> import sklearn
>>> sklearn.decomposition.PCA
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'sklearn' has no attribute 'decomposition'
>>>
```

This was also reported by Alex in #108
